test262/index.html

```diff
@@ -141,14 +141,28 @@ <h2 class="card-title">Older versions</h2>
       <div class="row">
         <div class="col-md-12" id="progress-info"></div>
       </div>
-      <div class="row">
-        <div class="col-md-12 d-none" id="info-options">
-          <div class="form-check form-switch">
+      <form class="d-none" id="info-options">
+        <div class="form-row d-flex">
+          <div class="form-group">
+            <select class="custom-select" id="info-options-es-version">
```

Comment:
Why `custom-select` here? I thought Bootstrap had `form-select` to enhance the look of the selection box:

https://getbootstrap.com/docs/5.3/forms/select/

Reply:
Done!